### PR TITLE
SLE15 add use_kerberos_security_all_exports rule to HIPAA profile

### PR DESCRIPTION
**Pull request by teacup-on-rockingchair**
#### Description:

Add use_kerberos_security_all_exports to SLE15 HIPAA profile

#### Rationale:

Add ansible,bash remediation and tests for use_kerberos_security_all_exports

### CODE REVIEW

File: linux_os/guide/services/nfs_and_rpc/nfs_configuring_servers/use_kerberos_security_all_exports/bash/shared.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# complexity = low
+# disruption = low
+
+nfs_exports=()
+readarray -t nfs_exports < <(grep -E "^/.*[[:space:]]+ .*\(.*\)[[:space:]]*$" /etc/exports | awk '{print $2}')
+
+for nfs_export in "${nfs_exports[@]}"
+do
+    correct_export=""
+    if [ "$(grep -c "sec=" <<<"$nfs_export")" -eq 0 ]; then
```
Comment (jan-cerny) [inline linux_os/guide/services/nfs_and_rpc/nfs_configuring_servers/use_kerberos_security_all_exports/bash/shared.sh:13]: You probably swap the command in the `if` branch with the command in the `else` branch. If the count of `sec=` in the string is 0, it won't make sense to try replacing `sec=` by sed.

**Review by jan-cerny — CHANGES_REQUESTED** [review]

File: linux_os/guide/services/nfs_and_rpc/nfs_configuring_servers/use_kerberos_security_all_exports/tests/no_sec_export.fail.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# platform = multi_platform_all
+
```
[the remark this answers is not in the document]
Comment (teacup-on-rockingchair) [reply linux_os/guide/services/nfs_and_rpc/nfs_configuring_servers/use_kerberos_security_all_exports/tests/no_sec_export.fail.sh:2]: Thx, obviously there was some bad copy-pasting from my test machine, should be fixed in a59ed41

File: linux_os/guide/services/nfs_and_rpc/nfs_configuring_servers/use_kerberos_security_all_exports/bash/shared.sh (hunk 1 above)
Comment (teacup-on-rockingchair) [reply linux_os/guide/services/nfs_and_rpc/nfs_configuring_servers/use_kerberos_security_all_exports/bash/shared.sh:13]: should be fixed in a59ed41

File: linux_os/guide/services/nfs_and_rpc/nfs_configuring_servers/use_kerberos_security_all_exports/bash/shared.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        correct_export="$(echo $nfs_export|sed  -e 's/sec=[^\,\)]*/sec=krb5\:krb5i\:krbp5/')"
```
[the remark this answers is not in the document]
Comment (teacup-on-rockingchair) [reply linux_os/guide/services/nfs_and_rpc/nfs_configuring_servers/use_kerberos_security_all_exports/bash/shared.sh]: Should be fixed in should be fixed in a59ed41

**Review by jan-cerny — APPROVED** [review]

**Pull request merged**